sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/SimpleDataType.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+/**
+ * Simple DataType is used to create SimpleSearchField.
+ */
+public class SimpleDataType extends ExpandableStringEnum<SimpleDataType> {
```

Comment:
You can't use the same one as for the convenience type? Less chance of breaking across generations.